packages/cheqd-sdk/src/dids/didCheqdUtil.ts

```diff
@@ -0,0 +1,110 @@
+import type { DidDocument } from '@aries-framework/core'
+import type { CheqdNetwork, DIDDocument, MethodSpecificIdAlgo, TVerificationKey } from '@cheqd/sdk'
+
+import {
+  createDidPayload,
+  createDidVerificationMethod,
+  createVerificationKeys,
+  DIDModule,
+  VerificationMethods,
+} from '@cheqd/sdk'
+import { MsgCreateDidDocPayload, MsgDeactivateDidDocPayload } from '@cheqd/ts-proto/cheqd/did/v2'
+import { bases } from 'multiformats/basics'
```

Comment:
The `multiformats` package has caused issues in react native in the past due to it's dependency on `TextEncoder`. Do we need this?  We have a base58 encoder implementation availalbe in AFJ (see TypedArrayEncoder). We also have a way to parse multibase keys in the `Key` class (`Key.fromFingerprint()`)

Reply:
This worked